tests/testthat/test-compute_age_years.R

```diff
@@ -0,0 +1,67 @@
+## Test 1: compute_age_years() works as expected when `age_unit` is a string ----
+test_that("compute_age_years Test 1: compute_age_years() works as expected when
+          `age_unit` is a string", {
```

Comment:
Please avoid that the test description uses more than one line. Otherwise, the "Format test_that test file" addin does not work.

Reply:
Done.